scripts/src/server.sh

```diff
@@ -130,6 +132,14 @@ if [ -z "$CONTAINER_NAME" ]; then
   CONTAINER_NAME="${CONTAINER_NAME/:/-}-ssh"
 fi
 
+if [[ ! -f "${SSH_KEY_FILE}" ]]; then
+  if [[ ! -f "${DEFAULT_SSH_RSA}" ]]; then
+    echo "SSH not set up! Configure SSH on your system."
+    exit 1
+  fi
+  echo "Provided SSH key file ${SSH_KEY_FILE} does not exist."
+  exit 1
```

Comment:
Wait why do you exit here? Shouldn't you do `${SSH_KEY_FILE} = ${DEFAULT_SSH_RSA}` instead

Reply:
right yes indeed good catch